coderd/coderd.go

```diff
@@ -1002,6 +1002,9 @@ func New(options *Options) *API {
 						})
 					})
 				})
+				r.Route("/paginated-members", func(r chi.Router) {
+					r.Get("/", api.paginatedMembers)
+				})
```

Comment:
Really unfortunate we cannot just use the `/members` endpoint without it being a breaking change.

Since organizations are new, I think it's worth considering.

Reply:
I was originally going to to augment the existing route to be paginated but @aslilac brought up the issue of backwards compatibility. I can circle back with her, and other stakeholders on this

Reply:
if we were having this discussion like two weeks ago I think I'd agree with you @Emyrk, but 2.20 is out now and marketing organizations as stable. :\